src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/RazorFileKind.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Razor.Language;
+
+public enum RazorFileKind : byte
+{
+    None = 0,
+    Component = 1,
+    ComponentImport = 2,
+    Legacy = 3
```

Comment:
Consider adding doc comments detailing that Component is `.razor`, ComponentImport is `_Imports.razor`, and Legacy is `.cshtml`.

Reply:
Good suggestion. Thanks!